### PR TITLE
Navigation Component

### PR DESCRIPTION
**Pull request by sahiba0915**
Files to check:

1. Navigation.html
2. Navigation.css

Live Preview: https://navcomp.netlify.app/component/navigation/navigation.html

### CODE REVIEW

**Review by akash-1318 — COMMENTED** [review]
Went through your navigation file looks good the way you have used form around input in header.

File: component/text/text-utility.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  </head>
+  <body>
+    <!-- NavBar -->
+
+    <nav class="navigation container">
+      <i class="nav-icon fab fa-uikit"></i>
+      <div class="nav-brand">UiPlugIn</div>
+      <ul class="list-non-bullet nav-pills">
+        <li class="list-item-inline">
+          <a class="link" href="/"_blank">Home</a>
```
Comment (chanchal-panpaliya) [inline component/text/text-utility.html:27]: href="/"_blank" -- ??? didn't understand

File: component/navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
+* {
```
Comment (chanchal-panpaliya) [inline component/navigation/navigation.css:1]: add root file in - index.css  then import link - import url(././.../indexfile_path)

File: component/navigation/navigation.html [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/atom-one-dark.min.css"
```
Comment (chanchal-panpaliya) [inline component/navigation/navigation.html:13]: import this file in index.css eg import url("http://........") .

**Review by chanchal-panpaliya — COMMENTED** [review]
code looks good

File: component/text/text-utility.html (hunk 1 above)
Comment (sahiba0915) [reply component/text/text-utility.html:27]: will correct that thanks that was by mistake i wanted to give target attribute.

File: component/navigation/navigation.css (hunk 2 above)
Comment (sahiba0915) [reply component/navigation/navigation.css:1]: ok will do that

File: component/navigation/navigation.html (hunk 3 above)
Comment (sahiba0915) [reply component/navigation/navigation.html:13]: ok

**Pull request merged**